scripts/check_changelog.js

```diff
@@ -0,0 +1,88 @@
+const { execSync } = require('child_process');
+const path = require('path');
+const fs = require('fs');
+
+// Colors for output
+const colors = {
+	red: text => `\x1b[31m${text}\x1b[0m`,
+	green: text => `\x1b[32m${text}\x1b[0m`,
+};
+
+try {
```

Comment:
@whitemoshui 
What about adding a check on the commit message, such that if it contains something like `--skip changelog`, then the returns without checking?
This way if someone wants to explicity avoid the changelog check, he/she will add `--skip changelog` to end of his commit message.

Reply:
let me have a try